dolphinscheduler-ui/src/js/conf/home/pages/security/pages/users/_source/createUser.vue

```diff
@@ -84,7 +84,7 @@
           </template>
         </m-list-box-f>
         <m-list-box-f>
-          <template slot="name">{{$t('Phone')}}</template>
+          <template slot="name"><strong>*</strong>{{$t('Phone')}}</template>
```

Comment:
Hi,
There is the phone judge, so we do need to set the photo blank is required.
You can just need to change the sql to `INSERT INTO `t_ds_user` VALUES ('1', 'admin', '7ad2410b2f4c074479a8937a28a22b8f', '0', 'xxx@qq.com', '', '0', '2018-03-27 15:48:50', '2018-10-24 17:40:22', null, 1);`
![image](https://user-images.githubusercontent.com/29545877/88470373-1bcbae80-cf2e-11ea-8fb8-95ca113b3db9.png)

Reply:
I think it is also necessary to add `<strong>*</strong>`  to remind users of this required item

Reply:
I have modified the SQL file,

Reply:
> I have modified the SQL file,

Hi~
If the mobile phone information is not currently used, there is no need to set photo as a required field.
And we just need to change the sql to [1], and not need to insert an example photo number in sql [2].
![image](https://user-images.githubusercontent.com/29545877/88470528-f6d83b00-cf2f-11ea-8155-d0a12a68697b.png)
![image](https://user-images.githubusercontent.com/29545877/88470508-b678bd00-cf2f-11ea-88e6-ab4d433190c8.png)

Reply:
OK, I'll revise it.

Reply:
I have revised and submitted.

Reply:
> I have revised and submitted.

Good job.
I think we also need to revert the <Strong> changes.

Reply:
Do you want to remove `<strong>*</strong>`?

Reply:
> Do you want to remove `<strong>*</strong>`?

Hi,
I think we do not need add `<strong>*</strong>`, because the phone is not required field, and `<strong>*</strong>` means required field.

Reply:
I have deleted and submitted.